generated/nifgen/session.py

```diff
@@ -3255,22 +3369,24 @@ def _error_message(self, error_code):
         return error_message_ctype.value.decode(self._encoding)
 
 
-class _RepeatedCapability(_SessionBase):
-    '''Allows for setting/getting properties and calling methods for specific repeated capabilities (such as channels) on your session.'''
-
-    def __init__(self, vi, repeated_capability):
-        super(_RepeatedCapability, self).__init__(repeated_capability)
-        self._vi = vi
-        self._is_frozen = True
-
-
 class Session(_SessionBase):
     '''An NI-FGEN session to a National Instruments Signal Generator.'''
 
     def __init__(self, resource_name, reset_device=False, option_string=""):
         super(Session, self).__init__(repeated_capability='')
+        self._library = library_singleton.get()
+        self._encoding = 'windows-1251'
         self._vi = 0  # This must be set before calling _initialize_with_channels().
         self._vi = self._initialize_with_channels(resource_name, reset_device, option_string)
+        self.channels = _Channels(self._vi, self._library, self._encoding)
+        self.p2p_streams = _P2PStreams(self._vi, self._library, self._encoding)
+        self.script_triggers = _ScriptTriggers(self._vi, self._library, self._encoding)
+        self.markers = _Markers(self._vi, self._library, self._encoding)
+        param_list = []
```

Comment:
This param_list stuff is just for __repr__ right? (this just for clarity)
Regardless consider moving the code that creates that script to a separate function or at least add a comment.

Reply:
Added comment since that is easier that code genning another function and code genning the call to the function.